docs/isoseq-tags.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Isoseq Tags
```

Comment:
1. It's `Iso-Seq`
2. Just call the page `BAM Tags`

Reply:
Updated to Iso-seq, BAM Tags.